tool.go

```diff
@@ -750,9 +750,31 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 	}
 
 	if runtime.GOOS != "windows" {
-		allArgs = append(allArgs, "--stack_size=10000", script)
+		// For non-windows OS environments, we've seen issues with stack space
```

Comment:
/s/windows/Windows/ since it's a brand name.

Also get rid of "OS", "For non-Windows environments" sounds better I think.